google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -1730,6 +1988,168 @@ public TransactionManager transactionManager(TransactionOption... options) {
     }
   }
 
+  class MultiplexedSession implements CachedSession {
+    SessionImpl delegate;
```

Comment:
This should be final, as we do not allow replacing.

Reply:
Done